Magento_Swatches/templates/product/layered/renderer.phtml

```diff
@@ -1,25 +1,27 @@
 <?php $swatchData = $block->getSwatchData(); ?>
-<div
+<ul
     attribute-code="<?= $swatchData['attribute_code'] ?>"
     attribute-id="<?= $swatchData['attribute_id'] ?>"
-    class="filter__swatch-wrapper"
+    class="list filter__list"
 >
     <?php foreach ($swatchData['options'] as $option => $label): ?>
         <?php if (isset($swatchData['swatches'][$option]['type'])): ?>
             <?php switch ($swatchData['swatches'][$option]['type']) {
                 case '3':
                 ?>
-                    <a
-                        href="<?= $label['link'] ?>"
-                        aria-label="<?= $label['label'] ?>"
-                    >
-                        <div
-                            tabindex="-1"
-                            option-type="3"
-                            option-id="<?= $option ?>"
-                            option-label="<?= $label['label'] ?>"
-                        ></div>
-                    </a>
+                    <li>
+                        <a
+                            href="<?= $label['link'] ?>"
+                            aria-label="<?= $label['label'] ?>"
+                        >
+                            <div
+                                tabindex="-1"
```

Comment:
I'm not sure if it's needed. div element shouldn't be focusable by default, additionally, the parent element a is focusable so if you don't set focus on this div (using js `.focus()` method, you can remove tabindex.

Reply:
tabindex removed, it's default magento code, I only changed the structure to `ul > li > a` in this file